acra/configuring-maintaining/general-configuration/acra_server.md

```diff
@@ -0,0 +1,419 @@
+---
+title: AcraServer
+bookCollapseSection: true
+weight: 3
+---
+
+# AcraServer
+
+## Command line flags
+
+* `--acracensor_config_file=<filename>`
+
+  Path to AcraCensor configuration file.
+
+* `--acraconnector_tls_transport_enable={true|false}`
+
+  Enable TLS to encrypt transport between AcraConnector and AcraServer.
+  Default is `false` which means "use SecureSession instead".
```

Comment:
This kitchensink of flags can be subdivided further:

* Network (`--db_host`, `--incoming_connection`)
* Keystore (`--auth_keys`, `--keys_dir`, `--keystore`)
* Caching
* MySQL configuration
* PostgreSQL configuration
* AcraConnector stuff
* AcraServer features (`--acrastruct_injectedcell_enable`, `--zonemode_enable`)
* whatever else you can see

Ditto for other binaries.

Reply:
`Network`, `Keystore` — done.
`Caching` — what except `--keystore_cache_size` (that already belongs to `Keystore`) is about caching?
`MySQL`, `PostgreSQL` — done, but they only got one flag to enable mysql, one to enable pgsql, two deprecated flags for pgsql, do we need sections for these?

BTW do we need to move Redis flags to `Keystore`? Since it is used as key DB. Or is it more about caching?